src/views/virtualmachines/details/tabs/details/hooks/useSSHService.tsx

```diff
@@ -1,37 +1,25 @@
 import * as React from 'react';
 
 import { ServiceModel } from '@kubevirt-ui/kubevirt-api/console';
+import { IoK8sApiCoreV1Service } from '@kubevirt-ui/kubevirt-api/kubernetes';
 import { V1VirtualMachineInstance } from '@kubevirt-ui/kubevirt-api/kubevirt';
-import { K8sResourceCommon, useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import { k8sGet } from '@openshift-console/dynamic-plugin-sdk';
 
 type UseSSHServiceValues = {
-  sshService: any;
+  sshService: IoK8sApiCoreV1Service;
 };
 
 const useSSHService = (vmi: V1VirtualMachineInstance): UseSSHServiceValues => {
-  const [sshService, setSSHService] = React.useState<any>(undefined);
-  const sshServiceModal = React.useMemo(
-    () => ({
-      kind: ServiceModel.kind,
-      isList: true,
-      namespace: vmi?.metadata?.namespace,
-    }),
-    [vmi],
-  );
-
-  const [services, isServicesLoaded] = useK8sWatchResource<K8sResourceCommon[]>(sshServiceModal);
```

Comment:
why not using useK8sWatchResource to get services? why u using k8sGet, you won't be updated with services changes..